### PR TITLE
[15.0][MIG] mrp_subcontracting_purchase_link: Migration to 15.0

### PR DESCRIPTION
**Pull request by MarinaAForgeFlow**
Standard migration to v15.

### CODE REVIEW

File: mrp_subcontracting_purchase_link/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "Link Purchase Order Line to Subcontract Productions",
+    "version": "15.0.1.0.0",
+    "category": "Manufacturing",
+    "license": "LGPL-3",
+    "author": "Quartile Limited, ForgeFlow, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/manufacture",
+    "depends": ["purchase", "mrp_subcontracting"],
```
Comment (MiquelRForgeFlow) [inline mrp_subcontracting_purchase_link/__manifest__.py]:
```suggestion
    "depends": ["mrp_subcontracting_purchase"],
```

**Review by MiquelRForgeFlow — COMMENTED** [review]
LGTM

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by JordiMForgeFlow — APPROVED** [review]
Code + functional LGTM 👍🏼

**Review by LoisRForgeFlow — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**